{
  "id": "EUVD-2020-0160",
  "enisaUuid": "f2bea75c-ad91-34fb-bdeb-5567d515817f",
  "description": "A vulnerability was discovered in the PyYAML library in versions before 5.3.1, where it is susceptible to arbitrary code execution when it processes untrusted YAML files through the full_load method or with the FullLoader loader. Applications that use the library to process untrusted input may be vulnerable to this flaw. An attacker could use this flaw to execute arbitrary code on the system by abusing the python/object/new constructor.",
  "datePublished": "Mar 24, 2020, 3:15:00 PM",
  "dateUpdated": "Mar 26, 2021, 2:44:00 AM",
  "baseScore": 9.8,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
  "references": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1747",
    "https://github.com/yaml/pyyaml/pull/386",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WORRFHPQVAFKKXXWLSSW6XKUYLWM6CSH/",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ZBJA3SGNJKCAYPSHOHWY3KBCWNM5NYK2/",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/K5HEPD7LEVDPCITY5IMDYWXUMX37VFMY/",
    "http://lists.opensuse.org/opensuse-security-announce/2020-04/msg00017.html",
    "http://lists.opensuse.org/opensuse-security-announce/2020-05/msg00017.html",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MMQXSZXNJT6ERABJZAAICI3DQSQLCP3D/",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/7PPAS6C4SZRDQLR7C22A5U3QOLXY33JX/",
    "https://github.com/advisories/GHSA-6757-jp84-gxfx",
    "https://nvd.nist.gov/vuln/detail/CVE-2020-1747",
    "https://github.com/github/advisory-database/pull/4942",
    "https://github.com/yaml/pyyaml/commit/0cedb2a0697b2bc49e4f3841b8d4590b6b15657e",
    "https://github.com/yaml/pyyaml/commit/5080ba513377b6355a0502104846ee804656f1e0",
    "https://github.com/pypa/advisory-database/tree/main/vulns/pyyaml/PYSEC-2020-96.yaml",
    "https://www.oracle.com/security-alerts/cpujul2022.html",
    "https://www.suse.com/security/cve/CVE-2020-1747.html",
    "https://access.redhat.com/errata/RHSA-2020:4641",
    "https://advisories.mageia.org/CVE-2020-1747.html",
    "https://linux.oracle.com/cve/CVE-2020-1747.html",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/WORRFHPQVAFKKXXWLSSW6XKUYLWM6CSH/",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/ZBJA3SGNJKCAYPSHOHWY3KBCWNM5NYK2/",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/K5HEPD7LEVDPCITY5IMDYWXUMX37VFMY/",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/MMQXSZXNJT6ERABJZAAICI3DQSQLCP3D/",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/7PPAS6C4SZRDQLR7C22A5U3QOLXY33JX/"
  ],
  "aliases": [
    "CVE-2020-1747",
    "GHSA-6757-jp84-gxfx",
    "PYSEC-2020-96"
  ],
  "assigner": "redhat",
  "epss": 2.59,
  "enisaIdProduct": [
    {
      "id": "b917c81f-f4d8-3d7b-93f0-55e8fdbfb8c1",
      "product": {
        "name": "PyYAML"
      },
      "product_version": "5.3.1"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "fa8eb730-0515-3e01-a0be-bc0fdc6c29e9",
      "vendor": {
        "name": "Red Hat"
      }
    }
  ]
}